### PR TITLE
tests: skip LAC tests

### PR DESCRIPTION
**Pull request by katbyte**

### CODE REVIEW

File: .teamcity/components/build_config_service.kt [hunk 1]
```diff
@@ -5,7 +5,7 @@ class serviceDetails(name: String, displayName: String, environment: String) {
     val displayName = displayName
     val environment = environment
 
-    fun buildConfiguration(providerName : String, nightlyTestsEnabled: Boolean, startHour: Int, parallelism: Int) : BuildType {
+    fun buildConfiguration(providerName : String, nightlyTestsEnabled: Boolean, startHstartHourour: Int, parallelism: Int) : BuildType {
```
Comment (mbfrahry) [inline .teamcity/components/build_config_service.kt:8]: Should this have changed?

**Review by mbfrahry — APPROVED** [review]
LGTM with one minor comment

File: .teamcity/components/build_config_service.kt (hunk 1 above)
Comment (katbyte) [reply .teamcity/components/build_config_service.kt:8]: probably not, weird

**Review by WodansSon — APPROVED** [review]
LGTM! 🚀

**Pull request merged**